src/cdn_definitions/schema.yaml

```diff
@@ -264,5 +273,6 @@ requiredProperties:
 - rhui_product_id
 - ignore_lp_version_product_ids
 - exclude_from_listings
+- rhel_open_dist
```

Comment:
Oops, this is not actually backwards-compatible.  I don't think we should be adding new properties to requiredProperties.  If we do, then upgrading cdn-definitions can suddenly change an existing data set from valid to invalid, which is a pretty strong discouragement from using the schema.

I guess the same mistake was made when the above fields were added and it was missed then, but now that this is live in cdn-utils, we should try harder to ensure it remains backwards-compatible.

Could you please drop this from requiredProperties?  I've also filed RHELDST-5670 about adding tests so an issue like this would be caught by CI.

Reply:
Thanks. I removed `rhel_open_dist` from `requiredProperties`.